src/pip/_internal/cli/cmdoptions.py

```diff
@@ -367,9 +367,13 @@ def find_links():
         action='append',
         default=[],
         metavar='url',
-        help="If a url or path to an html file, then parse for links to "
-             "archives. If a local path or file:// url that's a directory, "
-             "then look for archives in the directory listing.",
+        help="If a URL or path to an html file, then parse for links to "
+             "compressed archives such as source tree sdist (.tar.gz)  "
```

Comment:
"source tree sdist" is redundant - "sdist" is sufficient. (As this is the command line help, too, it needs to be kept concise). On the same basis, leave out "compressed".